src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * (radius * radius);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
+                + getArea() + " sq. units, radius: "
+                + radius + " units, color: "
+                + color);
+    }
+}
```

Comment:
The class Circle should not start with an empty line. Please remove the empty line at the beginning of the class definition.